docs/developers/setup.md

```diff
@@ -42,91 +42,40 @@ Here, we are running Ganache:
 
-When using UMA infrastructure, you often have to open the `truffle console` or run a script using `truffle exec`. When
-using truffle in the context of this repository, it's important to understand what types of keys you can use and what
-to specify for the `--network` argument.
+When using UMA infrastructure, you often have to open the Hardhat console using `npx hardhat console` or run a script using `npx hardhat run <script>`. When using Hardhat in the context of this repository, it's important to understand what to specify for the `--network` argument and how to set up your `hardhat.config.js` file.
 
 ### Public Networks
 
-Public networks include the Ethereum mainnet and any public testnets, like Rinkeby, Kovan, or Ropsten. If you are using
-these networks, you'll generally need to know:
+Public networks include the Ethereum mainnet and any public testnets, like Rinkeby, Kovan, or Ropsten. A list of UMA supported networks is available [here](https://github.com/UMAprotocol/protocol/blob/master/packages/common/src/PublicNetworks.ts).
 
-1. Which public network you intend to use.
-   A list of UMA supported networks is available [here](https://github.com/UMAprotocol/protocol/blob/master/packages/common/src/PublicNetworks.ts).
+The `--network` parameter is used in Hardhat commands to connect to a specific network. Generally, the network argument is structured as `--network [NETWORK_NAME]`. 
 
-2. What private key you want to use and how your private keys are stored.
-   Below is information regarding the two types of keys that are supported, mnemonics/seed phrases and hardware wallets.
-
-The `--network` parameter that's passed to all truffle commands depends on both of these factors. Here's an example:
-
-```bash
-yarn truffle console --network rinkeby_mnemonic
-```
-
-That command will tell truffle that the user wants to use the Rinkeby testnet and their private key is a mnemonic, or
-seed phrase. Generally, the network argument is structured as `--network [NETWORK_NAME]_[KEY_TYPE]`.
-
-### Mnemonic, or seed phrase (less secure)
-
-Mnemonics are much less secure than using a hardware wallet, but they are also much faster when sending multiple
-transactions since they don't require as much user input. If you'd like to use a mnemonic, you'll need to start by
-putting the mnemonic in your shell environment. Do this by running the following command:
-
-```bash
-export MNEMONIC="YOUR_MNEMONIC_HERE"
-```
-
-With a real mnemonic, this would look like:
-
-```bash
-export MNEMONIC="candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
-```
-
-Once you've done that you're ready to run a truffle command. When using a mnemonic, your network argument should look
-like `--network [NETWORK_NAME]_mnemonic`. So, for example, using a mnemonic on Kovan would look like:
+Here's an example of how to tell Hardhat to use the Rinkeby testnet:
 
 ```bash
-yarn truffle console --network kovan_mnemonic
+npx hardhat console --network rinkeby
```

Comment:
Do we use rinkeby for anything? Might be better to just give examples with Kovan

Reply:
Updated to use Kovan.